src/overlays/actors/ovl_En_Demo_heishi/z_en_demo_heishi.h

```diff
@@ -8,11 +8,25 @@ struct EnDemoheishi;
 typedef void (*EnDemoheishiActionFunc)(struct EnDemoheishi*, GlobalContext*);
 
 typedef struct EnDemoheishi {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x110];
-    /* 0x0254 */ EnDemoheishiActionFunc actionFunc;
-    /* 0x0258 */ char unk_258[0x68];
-} EnDemoheishi; // size = 0x2C0
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[17];
+    /* 0x1EE */ Vec3s morphTable[17];
+    /* 0x254 */ EnDemoheishiActionFunc actionFunc;
+    /* 0x258 */ s16 headRotY;
+    /* 0x25A */ s16 headRotX;
+    /* 0x25C */ s16 headRotZ;
+    /* 0x25E */ s16 headRotYTarget;
+    /* 0x260 */ s16 headRotXTarget;
+    /* 0x262 */ s16 headRotZTarget;
+    /* 0x264 */ s32 animIndex;
+    /* 0x268 */ s16 unk268;                         /* inferred */
+    /* 0x26A */ char pad26A[2];                     /* inferred */
+    /* 0x26C */ f32 frameCount;
+    /* 0x270 */ s16 unk270;                         /* inferred */
+    /* 0x272 */ s16 unk272;                         /* inferred */
```

Comment:
```suggestion
    /* 0x268 */ s16 unk268;
    /* 0x26C */ f32 frameCount;
    /* 0x270 */ s16 unk270;
    /* 0x272 */ s16 unk272;
```
These inferred comments leftover from mips2c don't really help us at this point we can probably remove them

And that pad is probably implicit, created by C aligning values inside of the struct, so it can be removed.

Reply:
If it can't be removed I would say change it to `UNK_TYPE1 unk26A;`

Reply:
Removed, thanks for the suggestion